### PR TITLE
Create py2 test target for internals.sendemail_test

### PR DESCRIPTION
**Pull request by KyleJu**
- Rename internals/sendemail_test.py to internals/sendemail_py2test.py and exclude it from `Continuous Integration / python_tests (pull_request)`

- Create a separate python2 test target on CI

- Duplicate the testing_config.py file and remove `ndb` dependencies for the py2 test

### CODE REVIEW

File: .github/workflows/ci_py2.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+jobs:
+  python2_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set up Python2
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.8'
```
Comment (KyleJu) [inline .github/workflows/ci_py2.yml:16]:
```suggestion
          python-version: '2.7'
```

File: testing_config_py2.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+import logging
+import os
+import sys
+import unittest
+
+app_engine_path = os.environ.get('APP_ENGINE_PATH', '')
+if not app_engine_path:
+  app_engine_path = '/usr/lib/google-cloud-sdk/platform/google_appengine'
+if not os.path.exists(app_engine_path):
```
Comment (KyleJu) [inline testing_config_py2.py:26]: Remove this after done

**Comment by KyleJu** [comment]
The `python2_tests` failure looks a lot like https://github.com/GoogleChrome/chromium-dashboard/runs/3635040030, the CI failure on main, which is a good sign.

```ERROR: internals.sendemail_py2test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: internals.sendemail_py2test
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/2.7.18/x64/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/opt/hostedtoolcache/Python/2.7.18/x64/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/internals/sendemail_py2test.py", line 26, in <module>
    from internals import models
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/internals/models.py", line 26, in <module>
    from google.cloud import ndb
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/lib/google/cloud/ndb/__init__.py", line 28, in <module>
    from google.cloud.ndb.client import Client
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/lib/google/cloud/ndb/client.py", line 24, in <module>
    from google.cloud import _helpers
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/lib/google/cloud/_helpers.py", line 33, in <module>
    from google.protobuf import duration_pb2
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/lib/google/protobuf/duration_pb2.py", line 5, in <module>
    from google.protobuf import descriptor as _descriptor
  File "/home/runner/work/chromium-dashboard/chromium-dashboard/lib/google/protobuf/descriptor.py", line 113
    class DescriptorBase(metaclass=DescriptorMetaclass):
                                  ^
SyntaxError: invalid syntax
```

**Review by jrobbins — APPROVED** [review]
OK, this setup looks like it can be made to work.

Right now sendemail.py does indirectly use models.py, but I realize now that I need to re-write it to have no shared code with rest of the app.  Instead, anything happening in py2 would only create a task, which is then routed to the py3 app to do any datastore interactions.

**Comment by KyleJu** [comment]
> OK, this setup looks like it can be made to work.
> 
> Right now sendemail.py does indirectly use models.py, but I realize now that I need to re-write it to have no shared code with rest of the app. Instead, anything happening in py2 would only create a task, which is then routed to the py3 app to do any datastore interactions.

Yea I briefly looked at `models.py`. It is py2/py3 compatible but the more `sendemail.py` is isolated, the better.

**Pull request merged**